tests/lcb/stdlib/java.lcb

```diff
@@ -40,7 +40,7 @@ end handler
 
 foreign handler CreateJavaString(in pBytes as JByteArray) returns JString binds to "java:java.lang.String>new([B)"
 
-foreign handler MCDataCreateWithBytes(in pBytes as ZStringNative, in pCount as CUInt, out rData as Data) returns CBool binds to "<builtin>"
+foreign handler MCDataCreateWithBytes(in pBytes as ZStringNative, in pCount as CUInt, out rData as Data) returns JBoolean binds to "<builtin>"
```

Comment:
This one's binding to the libfoundation function so should remain `CBool` shouldn't it?